crates/deterministic-wasi-ctx/Cargo.toml

```diff
@@ -11,12 +11,12 @@ categories = ["wasm"]
 [dependencies]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
-cap-primitives = "0.26.1"
-wasi-common = ">= 2.0.0, < 3.1.0"
+cap-primitives = "1.0.3"
+wasi-common = ">= 4.0.0, < 4.1.0"
```

Comment:
I don't think this change is backwards compatible below 4 so I updated the minimum supported version.